dotnet-estimation/dotnet-estimation/Templates/WebAPI/Devon4Net.Application.WebAPI.Implementation/Business/SessionManagement/Service/SessionService.cs

```diff
@@ -74,5 +99,13 @@ public async Task<Session> GetSession(long id)
 
             return (sessionIsValid, null);
         }
+
+        private string generateToken()
```

Comment:
Name sollte vielleicht in "generateInviteToken()" umbenannt werden, um verständiskeits konflikte bit den JWT webtoken aus Vlads teil zu verhindern.